src/frontend/js/widgets/Dashboard/components/DashboardCard/index.tsx

```diff
@@ -1,13 +1,24 @@
 import { PropsWithChildren, ReactNode, useMemo, useRef, useState } from 'react';
+import c from 'classnames';
 import { Button } from 'components/Button';
 import { Icon, IconTypeEnum } from 'components/Icon';
 
 interface Props {
   header: ReactNode | string;
   footer?: ReactNode;
+  expendable?: boolean;
+  fullwidth?: boolean;
+  className?: string;
 }
 
-export const DashboardCard = (props: PropsWithChildren<Props>) => {
+export const DashboardCard = ({
+  children,
+  header,
+  footer,
+  className,
+  expendable = true,
```

Comment:
I don't think "expendable" is what you mean ^^
```suggestion
  expandable = true,
```